components/list/list-item.js

```diff
@@ -72,6 +70,17 @@ class ListItem extends ListItemCheckboxMixin(ListItemRoleMixin(RtlMixin(LitEleme
 			:host(:last-child) d2l-list-item-generic-layout[data-separators="between"] {
 				border-bottom: 1px solid transparent;
 			}
+			:host([dragging]) d2l-list-item-generic-layout {
+				filter: grayscale(75%);
+				opacity: 0.4;
+			}
+			.d2l-list-item-drag-image {
+				transform: rotate(-1deg);
+			}
+			d2l-list-item-generic-layout {
+				background: white;
+				transform: rotate(1deg);
```

Comment:
We needed tilt. Tilt that supported images. This, this was the best solution. 
`background: white` is required so the drag image isn't transparent.